cg-spec/editors_draft.html

```diff
@@ -895,6 +895,47 @@ <h2>Grammar</h2>
             <a data-cite="SPARQL11-QUERY#rTriplesNodePath">TriplesNodePath</a>
           </td>
         </tr>
+        <tr id="rPrimaryExpression">
+          <td>[119]</td>
+          <td>`PrimaryExpression`</td>
+          <td>::=</td>
+          <td>
+            <a data-cite="SPARQL11-QUERY#rBrackettedExpression">BrackettedExpression</a> `|` 
+            <a data-cite="SPARQL11-QUERY#rBuiltInCall">BuiltInCall</a> `|` 
+            <a data-cite="SPARQL11-QUERY#ririOrFunction">iriOrFunction</a> `|` 
+            <a data-cite="SPARQL11-QUERY#rRDFLiteral">RDFLiteral</a> `|` 
+            <a data-cite="SPARQL11-QUERY#rNumericLiteral">NumericLiteral</a> `|`
+            <a data-cite="SPARQL11-QUERY#rBooleanLiteral">BooleanLiteral</a> `|`
+            <a data-cite="SPARQL11-QUERY#rVar">Var</a>
+            <div style="font-weight: bold">
+            `|` <a href="#rExprEmbTP" style="font-weight: bold">ExprEmbTP</a>
+            </div>
+          </td>
+        </tr>
+        <tr id="rExprEmbTP">
+          <td>[119a]</td>
+          <td style="font-weight: bold">`ExprEmpTP`</td>
+          <td style="font-weight: bold">::=</td>
+          <td style="font-weight: bold">
+            <code class="token">'&lt;&lt;'</code>
+            <a href="#rExprVarOrTerm">ExprVarOrTerm</a>
+            <a data-cite="SPARQL11-QUERY#rVerb">Verb</a>
+            <a href="#rExprVarOrTerm">ExprVarOrTerm</a>
+            <code class="token">'&gt;&gt;'</code>
+          </td>
+        </tr>
+        <tr id="rExprVarOrTerm">
+          <td>[119b]</td>
+          <td style="font-weight: bold">`ExprVarOrTerm`</td>
+          <td style="font-weight: bold">::=</td>
+          <td style="font-weight: bold">
+            <a data-cite="SPARQL11-QUERY#riri">iri</a> `|` 
+            <a data-cite="SPARQL11-QUERY#rRDFLiteral">RDFLiteral</a> `|` 
+            <a data-cite="SPARQL11-QUERY#rNumericLiteral">NumericLiteral</a> `|`
+            <a data-cite="SPARQL11-QUERY#rBooleanLiteral">BooleanLiteral</a> `|`
+            <a data-cite="SPARQL11-QUERY#rVar">Var</a>
+          </td>
```

Comment:
Should this include `ExprVarOrTerm` in the same way that `EmbSubjectOrObject` includes `EmbTP`? Or, is that not possible in this context?

Reply:
Done.